contrib/mhchem/README.md

```diff
@@ -0,0 +1,17 @@
+# mhchem extension
+
+This extension adds to KaTeX the `\ce` and `\pu` functions from the [mhchem](https://mhchem.github.io/MathJax-mhchem/) package.
+
+### Usage
+
+This extension isn't part of core KaTeX, so the script should be separately included. Write the following line into the page�s `<head>`. Place it *after* the line that points to `katex.js`.
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/katex@0.10.0-beta/dist/contrib/mhchem.min.js"></script>
```

Comment:
This will need to be updated and we'll need to do a publish for this to work.  Also, the webpack config files need to be updated to compile this file.